settings/scott/demographics.yml

```diff
@@ -39,7 +39,9 @@ demographics:
       haart:
         init: 1.0
         prob: 0.679
-        adherence: 1.0
+        adherence:
```

Comment:
it looks like all of these haart's are the same, I'd throw a `&base_haart` at the end of line 39, then replace the subsequent haart blocks with `haart: *base_haart` (I've done this a bunch over in the params PR for examples)